src/System.ComponentModel/tests/ComponentModelBasicTests.cs

```diff
@@ -2,49 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.ComponentModel;
 using Xunit;
 
-namespace Test
+namespace System.ComponentModel.Tests
 {
     public class ComponentModelTests
     {
         [Fact]
-        public static void TestComponentModelBasic()
+        public static void SystemComponentModel_Interfaces_LoadedSuccessfully()
         {
-            // dummy tests to make sure the System.ComponentModel library loaded successfully
-#pragma warning disable 0219
             IRevertibleChangeTracking iRevertibleChangeTracking = null;
             IChangeTracking iChangeTracking = iRevertibleChangeTracking;
             Assert.Null(iChangeTracking);
 
             IEditableObject iEditableObject = null;
-            CancelEventArgs cancelEventArgs = new CancelEventArgs();
-            Assert.NotNull(cancelEventArgs);
+            Assert.Null(iEditableObject);
```

Comment:
oh I suppose it's to force it to not optimize away. suggesting this test is some kind of setup for the other tests. i wonder if it can just be deleted.